wormhole-connect/package.json

```diff
@@ -109,7 +110,7 @@
     "vite-bundle-visualizer": "^0.11.0",
     "vite-plugin-checker": "^0.6.2",
     "vite-plugin-dts": "^3.7.3",
-    "vite-plugin-node-polyfills": "^0.17.0"
+    "@artursapek/vite-plugin-node-polyfills": "0.21.1"
```

Comment:
this is a fork which contains this fix https://github.com/davidmyersdev/vite-plugin-node-polyfills/pull/89/files